time stamp
float64 voltage
float64 vmax
float64 vmin
